source/includes/_appliance_part_type_replacement_linker.md

```diff
@@ -10,6 +10,7 @@ __appliance_part_type_replacement_linker_id__ <br><font color="DarkGray">_int_</
 __<a href="/#appliance-type">appliance_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
 __<a href="/#existing-appliance-part-type">existing_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
 __<a href="/#replacement-appliance-part-type">replacement_appliance_part_type_id</a>__ <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,foreign-key)</font> | 
+__max_part_count__  <br><font color="DarkGray">_int_</font> <font color="Crimson">(not-null,default=1)</font>| Count of appliance_part_types for each Appliance
```

Comment:
"Maximum number of parts of this type that can be replaced for this appliance type"